blog.md

```diff
@@ -47,7 +47,7 @@ Now that we have created the endpoints we can start with training the model.
 ### It all starts with a question 
 Every Machine Learning journey starts with a question you want to have answered. For this example, you are going to answer the question: Is it a Homer or a Marge Lego figure. 
```

Comment:
Might be worth to break up the text to add any open license images you can find of homer and marge?